docs/mining/proof-of-work.md

```diff
@@ -69,30 +69,21 @@ GPU drivers usually contain the libraries needed for mining.  If you have diffic
 
 ## **<i class="fa fa-download"></i> Select and Download Mining Software **
 
-> Official Decred Mining Software Builds
+### Official Decred Miner (gominer)
 
-#### **[<i class="fa fa-github"></i> gominer](https://github.com/decred/gominer)**
-
-It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **[<i class="fa fa-github"></i>v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
+It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **<i class="fa fa-github"></i> [v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
```

Comment:
Agreed the wording could be a bit softer. How about:

"Gominer is the official Decred miner developed and supported by the Decred team. It is the easiest miner to setup and start mining and so it is recommended for most users."